### PR TITLE
added new txt file and added changes in checkcpp file

### PR DESCRIPTION
**Pull request by antrap**
entered hello world txt by creating new text file and wrote a cpp code in checkcpp.cpp file.

### CODE REVIEW

File: checkcpp.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include<iostream>
+using namespace std;
+
```
Comment (bibhashM) [inline checkcpp.cpp:3]: remove extra lines between code

Comment (antrap) [reply checkcpp.cpp:3]: removed extra lines

**Review by bibhashM — COMMENTED** [review]
ok to go!!

**Review by bibhashM — COMMENTED** [review]
learning PR reviewed

**Review by bibhashM — APPROVED** [review]

**Pull request merged**